attribute/documentdetailsattribute.go

```diff
@@ -78,12 +73,12 @@ func NewDocumentDetails(a *yotiprotoattr.Attribute) (*DocumentDetailsAttribute,
 
 // Parse fills a DocumentDetails object from a raw string
 func (details *DocumentDetails) Parse(data string) (err error) {
-	if !documentDetailsValidateRegexp.MatchString(data) {
+	dataSlice := strings.Split(data, " ")
+
+	if len(dataSlice) < 3 {
 		return fmt.Errorf("Document Details data is invalid, %s", data)
 	}
```

Comment:
Can we add in a check to make sure that there are no empty strings in the split array?  I think this will also need tests which you can lift from the other SDK's

Reply:
Have added this check just below.